tools/test/sofcitest/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 sof-ci
```

Comment:
Why is this MIT licence ?. The Copyright needs to be Intel. Is this your own work or derived from somewhere else ?